sklearn/tests/test_metadata_routing.py

```diff
@@ -109,12 +109,18 @@ def record_metadata(obj, method, record_default=True, **kwargs):
     obj._records[method] = kwargs
 
 
-def check_recorded_metadata(obj, method, **kwargs):
+def check_recorded_metadata(obj, method, split_params=tuple(), **kwargs):
     """Check whether the expected metadata is passed to the object's method."""
```

Comment:
I think that it will start to be worth documented the parameter here.
For instance what is the meaning of `split_params`